pages/home-v2-wip.tsx

```diff
@@ -0,0 +1,16 @@
+import FeaturedMarkeplacesSection from '@/common/components/home/FeaturedMarketplacesSection';
+import Footer from '@/common/components/home/Footer';
+import { FC } from 'react';
+
+const Home: FC = () => {
+  return (
+    <>
+      <div className="container mx-auto pb-48 w-[80%]">
+        <FeaturedMarkeplacesSection />
+      </div>
+      <Footer />
+    </>
+  );
+};
+
+export default Home;
```

Comment:
Why even have a page component if its going to import everything from another component?

Also personally don't like things like HomeSection, HomeSection.Body. What I do like is shared layouts objects like Section, Row, Col.

Tailwind is easy enough to work with just use vanilla html elements. This is 17 files that could really be like 5. 

Not trashing on it just want to get my concerns across.

Reply:
> Why even have a page component if its going to import everything from another component?

@kristianeboe is right that this is a placeholder for the future v2 `index.tsx`

> ChevronRight could also be replaced entirely everywhere with an export from Heroicons/outline.

I believe I'm using the hero icons throughout my code. I will create a ticket for changing usages everywhere, though (#406).

> As for an explosion in number of components, maybe we can find a middle ground? IMO we should move code from HomeSection and HomePage back into the home-v2-wip page, but keep FeaturedMarketplacesSection and MarketplacePreview(Card).

Will do this. 

I disagree with the tailwind approach that everything should be flat and you should avoid wrapping stuff into components. While it's true I have wound up with some specialized component names, it's also true that repeated giant classNames and nested elements are hard to understand and maintain, given how complex and fragile markup is. I shouldnt need to check every `div` at the same level for a `hover` selector to know whether or not something happens when it is hovered, and that every `div` has the same hover behavior.

Reply:
Actually @kristianeboe if I want to get rid of `HomeSection` , all of that will move into `FeaturedMarketplacesSection`. I will then have to repeat the same markup in `FeaturedBuyNow`, `FeaturedProfiles`, etc. So, if we are getting rid of `HomeSection`, then IMO the whole thing should flatten into `home-v2-wip` as @kespinola suggests, so that the shared code can be seen in the same document.

Reply:
I moved everything into `home-v2-wip`, but also repeated some stuff in `HomeV1FeaturedMarketplacesSection` so I could include that (temporarily) in the existing homepage